test/vcr_python_wrapper.py

```diff
@@ -41,6 +41,21 @@ def domain_query_matcher(r1, r2):
     return r1.query == r2.query
 
 
+def host_body_matcher(r1, r2):
+    if r1.headers.get('content-type') == r2.headers.get('content-type') == 'application/json':
+        if r1.path == r2.path == '/api/v2/hostgroups':
+            r1_copy = vcr.request.Request(r1.method, r1.uri, r1.body, r1.headers)
+            r2_copy = vcr.request.Request(r2.method, r2.uri, r2.body, r2.headers)
+            body1 = json.loads(r1_copy.body.decode('utf8'))
+            body2 = json.loads(r2_copy.body.decode('utf8'))
+            body1['search'] = "name='test_group'"
```

Comment:
I actually meant to use `body1['search'] = 'name="test_group"'` in my previous comment (as `"` is what our code that generates that query will use), but I guess it doesn't matter which way round we quote in the overwritten version :)

Reply:
I would still try to produce fakes that _look right_.

Reply:
I will just change that to be ready to merge.